pyGPG/config.py

```diff
@@ -27,11 +27,19 @@ class GPGConfig(object):
     defaults = {
         'clearsign': '--clearsign',
         'decrypt': '--decrypt',
+<<<<<<< HEAD
+=======
+        'encrypt': '--encrypt',
+>>>>>>> e65b4e2... my version
         'delete-keys': '--delete-keys',
         'detach-sign': '--detach-sign',
         'dump-options': '--dump-options',
         'fingerprint': '--fingerprint',
+<<<<<<< HEAD
         'gpg_command': '/usr/bin/gpg',
+=======
+        'gpg_command': '/usr/pkg/bin/gpg',
```

Comment:
This looks to be distro specific.  What distribution of linux is this for?  It might be something that should be patched at the distro packaging level.  It also might be possible to auto-determine this.  But '/usr/pkg/bin/gpg' is definitely wrong for a Gentoo based install.